pkgs/data/icons/numix-cursor-theme/default.nix

```diff
@@ -1,28 +1,19 @@
-{ lib, stdenv, fetchFromGitHub, fetchpatch, inkscape, xcursorgen }:
+{ lib, stdenv, fetchFromGitHub, inkscape, xcursorgen }:
 
 stdenv.mkDerivation rec {
-  version = "1.1";
+  version = "1.2";
   package-name = "numix-cursor-theme";
   name = "${package-name}-${version}";
```

Comment:
```suggestion
  pname = "numix-cursor-theme";
  version = "1.2";
```

Reply:
Thank you for the review! I applied your suggestion.